dymos/transcriptions/pseudospectral/gauss_lobatto.py

```diff
@@ -422,236 +289,248 @@ def configure_path_constraints(self, phase):
-                              tgt_name='{0}.input_values:states:{1}'.format(name, state_name))
+                phase.connect(src_name=f'control_rates:{control_name}_rate2',
+                              tgt_name=f'{timeseries_name}.input_values:control_rates:{control_name}_rate2')
 
-                phase.connect(src_name='interleave_comp.all_values:state_rates:{0}'.format(state_name),
-                              tgt_name='{0}.input_values:state_rates:{1}'.format(name, state_name))
+            for control_name, options in phase.polynomial_control_options.items():
+                control_units = options['units']
 
-            for control_name, options in phase.control_options.items():
                 # Control values
-                phase.connect(src_name='control_values:{0}'.format(control_name),
-                              tgt_name='{0}.input_values:controls:{1}'.format(name, control_name))
-
-                # Control rates
-                phase.connect(src_name='control_rates:{0}_rate'.format(control_name),
-                              tgt_name='{0}.input_values:control_rates:{1}_rate'.format(name, control_name))
+                phase.connect(src_name=f'polynomial_control_values:{control_name}',
+                              tgt_name=f'{timeseries_name}.input_values:'
+                                       f'polynomial_controls:{control_name}')
 
-                # Control second derivatives
-                phase.connect(src_name='control_rates:{0}_rate2'.format(control_name),
-                              tgt_name='{0}.input_values:control_rates:{1}_rate2'.format(name, control_name))
-
-            for control_name, options in phase.polynomial_control_options.items():
-                # Control values
-                phase.connect(src_name='polynomial_control_values:{0}'.format(control_name),
-                              tgt_name='{0}.input_values:'
-                                       'polynomial_controls:{1}'.format(name, control_name))
+                timeseries_comp._add_output_configure(f'polynomial_controls:{control_name}',
+                                                      shape=options['shape'],
+                                                      units=control_units,
+                                                      desc=options['desc'])
 
                 # Control rates
-                phase.connect(src_name='polynomial_control_rates:{0}_rate'.format(control_name),
-                              tgt_name='{0}.input_values:'
-                                       'polynomial_control_rates:{1}_rate'.format(name, control_name))
+                phase.connect(src_name=f'polynomial_control_rates:{control_name}_rate',
+                              tgt_name=f'{timeseries_name}.input_values:'
+                                       f'polynomial_control_rates:{control_name}_rate')
+
+                timeseries_comp._add_output_configure(f'polynomial_control_rates:{control_name}_rate',
```

Comment:
I don't think I ever asked about the name `_add_output_configure`, but you could change that if you want. You figured out how to use it though, so that's the most important thing.